test/metabase/api/dashboard_test.clj

```diff
@@ -523,7 +523,34 @@
                  (into {} (for [[field-id m] response]
                             [field-id (update m :values (partial take 3))])))))))))
 
-
+(deftest dashboard-param-link-to-a-field-without-full-field-values-test
+  (testing "GET /api/dashboard/:id"
+    (t2.with-temp/with-temp [:model/Dashboard
```

Comment:
Subjective opinion but it will give you more space and will be easier to read if you put this on a new line
```suggestion
    (t2.with-temp/with-temp 
      [:model/Dashboard ...
```